src/lib/FeeEscalatorLib.sol

```diff
@@ -9,6 +9,17 @@ import {ReactorErrors} from "../base/ReactorErrors.sol";
 library FeeEscalatorLib {
     using FixedPointMathLib for uint256;
 
+    bytes internal constant FEE_ESCALATOR_TYPESTRING = abi.encodePacked(
+        "FeeEscalator(",
+        "address token,",
+        "uint256 startAmount,",
+        "uint256 endAmount,",
```

Comment:
previously you were skipping endAmount since it's included in the tokenpermissions - any reason we decided to add it back in?